### PR TITLE
Revise and resolve conflicts

### PR DESCRIPTION
**Pull request by jpmcfarland**

### CODE REVIEW

**Comment by jpmcfarland** [comment]
@chStaiger I added a missing edit and fixed a small bug.  The changes should be minor enough not to complicate things for you.

File: gui/mainmenu.py [hunk 1]
```diff
@@ -49,35 +48,48 @@ def __init__(self, widget, ic, ienv):
...
-                # Grant access by tickets, index 4
-                if 'tabCreateTicket' in ienv['ui_tabs']:
-                    self.createTicket = gui.irodsCreateTicket.irodsCreateTicket(ic, self.ienv)
-                    self.tabWidget.addTab(self.createTicket, "Create access tokens")
-            # General info
+                for tab in ienv["ui_tabs"]:
+                    if tab in ui_tabs_lookup:
+                        ui_tabs_lookup[tab](ic)
+                    else:
+                        logging.error("Unknown tab \"{uitab}\" defined in irods environment file".format(uitab=tab))
```
Comment (jpmcfarland) [inline gui/mainmenu.py:67]: Nice!  I had not thought to do this, but it is the kind of consolidation method I would have chosen.

**Comment by jpmcfarland** [comment]
I have gone through all the changes and everything looks good to me.

**Review by chStaiger — APPROVED** [review]
The main tabs all work for me:
Browser
Data Transfers
Info

also the experimental tab ElabJournal is ok.

Can be merged!!! Many thanks!

**Pull request merged**